Nos/Views/ReplyCountLabel.swift

```diff
@@ -0,0 +1,154 @@
+import Dependencies
+import SwiftUI
+
+struct ReplyCountLabel: View {
```

Comment:
I opened this file up hoping for a SwiftUI preview. Can you add one? Now that we have them working again I think we should get back to our convention of adding a preview for every view that isn't trivial.

(I tried previewing NoteButton and NoteCard as well but I couldn't see this view rendered there either.)

Reply:
Previews are still not working for me. I need to downgrade, but I can only do that once I'm back home due to network limitations (downloading Xcode). I've opened a ticket [here](https://github.com/planetary-social/nos/issues/1297) to ensure I don't forget about this.

Reply:
Did you try building for testing before rendering the preview? I thought they worked on the newer versions of xcode with that caveat.

Reply:
Yes, but they don't work for me at the moment unfortunately.